### PR TITLE
Update flow diagram

### PR DESCRIPTION
**Pull request by jerae-duffin**
Update flow diagram for SSO

### CODE REVIEW

**Review by ivan-californias — DISMISSED** [review]
👍 LGTM

**Review by technicallyjosh — DISMISSED** [review]
Thanks @ivan-californias 🎉

File: single-sign-on/index.md [hunk 1]
```diff
@@ -18,6 +18,14 @@ To enable SSO in Docker Hub, you need the following information from your identi
 
 We currently support enabling SSO on a single organization. However, we do not support single logout. If you have any users in your organization with a different domain (including social domains), they will be added to the organization as guests. Guests will continue to authenticate through Docker with their Docker login credentials (Docker ID and password).
 
+
+## Single Sign-on Architecture flow
+
+The following diagram shows how Single Sign-On (SSO) operates and is managed in Docker Hub and Docker Desktop. In addition, it provides information on how to authenticate between your IdPs.
```
Comment (craig-osterhout) [inline single-sign-on/index.md:24]:
```suggestion
The following diagram shows how Single Sign-on (SSO) operates and is managed in Docker Hub and Docker Desktop. In addition, it provides information on how to authenticate between your IdPs.
```

File: single-sign-on/index.md [hunk 2]
```diff
@@ -18,6 +18,14 @@ To enable SSO in Docker Hub, you need the following information from your identi
 
 We currently support enabling SSO on a single organization. However, we do not support single logout. If you have any users in your organization with a different domain (including social domains), they will be added to the organization as guests. Guests will continue to authenticate through Docker with their Docker login credentials (Docker ID and password).
 
+
+## Single Sign-on Architecture flow
```
Comment (craig-osterhout) [inline single-sign-on/index.md:22]: Noting that `Architecture` is capitalized here. It may be correct and intended. It just popped out at me.

**Review by craig-osterhout — APPROVED** [review]

**Comment by usha-mandya** [comment]
Thank you @jerae-duffin and @ivan-californias. Not sure if users can read any info in the diagram -  https://deploy-preview-15636--docsdocker.netlify.app/single-sign-on/#single-sign-on-architecture-flow. I'm looking at other options so see if we can allow users to zoom in to view the details.

File: single-sign-on/index.md (hunk 1 above)
Comment (jerae-duffin) [reply single-sign-on/index.md]: Thanks Craig, updated!

File: single-sign-on/index.md (hunk 2 above)
Comment (jerae-duffin) [reply single-sign-on/index.md]: Thank you! It should be LC

File: single-sign-on/index.md [hunk 3]
```diff
@@ -18,6 +18,14 @@ To enable SSO in Docker Hub, you need the following information from your identi
 
 We currently support enabling SSO on a single organization. However, we do not support single logout. If you have any users in your organization with a different domain (including social domains), they will be added to the organization as guests. Guests will continue to authenticate through Docker with their Docker login credentials (Docker ID and password).
 
+
+## Single Sign-on architecture flow
+
+The following diagram shows how Single Sign-on (SSO) operates and is managed in Docker Hub and Docker Desktop. In addition, it provides information on how to authenticate between your IdPs.
+
+![SSO architecture](images/sso-architecture.png)
```
Comment (usha-mandya) [inline single-sign-on/index.md:26]: @jerae-duffin The following syntax creates a zoomable version of the image. Tried this by cloning your PR branch and building docs locally on my machine. Let me know if this works for you.

```[SSO architecture](images/sso-architecture.png)```

Comment (usha-mandya) [reply single-sign-on/index.md]: Wondering if we can also include `{: target="_blank" rel="noopener" class="_"}` so the image opens in a new tab? Without this option, if a user clicks the `X`  button to close the zoomed in image, it closes the browser session

Comment (jerae-duffin) [reply single-sign-on/index.md]: Updated! Thanks so much!

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**